packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -254,6 +275,37 @@ switch (fruit) {
 </TabItem>
 </Tabs>
 
+And you can mark the default case as a comment.
+
+<Tabs>
+<TabItem value="❌ Incorrect">
+
+```ts option='{ "considerDefaultExhaustiveForUnions": true }
+declare const literal: 'a' | 'b' | 'c';
+
+switch (literal) {
+  case 'a':
+    break;
+  // no default
+}
+```
+
+</TabItem>
+<TabItem value="✅ Correct">
+
+```ts
+declare const value: 'a' | 'b';
+
+switch (value) {
+  case 'a':
+    break;
+  // no default
+}
+```
+
+</TabItem>
+</Tabs>
+
```

Comment:
[Docs] I don't think this is necessary, given that `defaultCaseCommentPattern` already documents itself and automatically mentions its default value.

```suggestion
```